client/homebrew/editor/editor.jsx

```diff
@@ -174,9 +178,42 @@ const Editor = createClass({
 		}
 	},
 
-	brewJump : function(){
-		const currentPage = this.getCurrentPage();
-		window.location.hash = `p${currentPage}`;
+	brewJump : function(targetPage=this.getCurrentPage()){
+		if(!window || this.isMeta()) return;
```

Comment:
Perhaps the buttons should not be present at all in the Meta tab (and `left` and `right` should not appear in the Style tab). Instead of checking here and having a nonfunctional button.

Reply:
SnippetBar buttons have been removed; Divider buttons only appear in Content Editor.